client/src/pages/locations/Form.js

```diff
@@ -60,6 +61,8 @@ const LocationForm = ({ edit, title, initialValues, notesComponent }) => {
   const { currentUser } = useContext(AppContext)
   const navigate = useNavigate()
   const [error, setError] = useState(null)
+  const [childFunctionTrigger, setChildFunctionTrigger] = useState(false)
+  const [relatedObjectUuid, setRelatedObjectUuid] = useState()
```

Comment:
Do you really need a `relatedObjectUuid` state field to get the location's uuid?